src/protocols/aeternity/views/AccountCreateModal.vue

```diff
@@ -39,18 +39,36 @@
         :disabled="!isOnline"
         @click="createMultisigAccount()"
       />
+      <BtnSubheader
+        v-if="!isMultisig"
+        :header="$t('modals.importAirGapAccount.btnText')"
+        :subheader="$t('modals.importAirGapAccount.btnSubtitle')"
+        :icon="QrScanIcon"
+        @click="connectHardwareWallet()"
+      />
     </div>
   </Modal>
 </template>
 
 <script lang="ts">
 import { defineComponent, PropType } from 'vue';
-import { MODAL_MULTISIG_VAULT_CREATE, PROTOCOLS } from '@/constants';
-import { useAccounts, useConnection, useModals } from '@/composables';
+import {
+  MODAL_AIR_GAP_IMPORT_ACCOUNTS,
+  MODAL_MULTISIG_VAULT_CREATE,
+  PROTOCOLS,
+} from '@/constants';
+import { IAirgapAccountRaw } from '@/types';
```

Comment:
```suggestion
import type { IAirgapAccountRaw } from '@/types';
```
...and please put this line before the constants :)